OBS-Update.md

```diff
@@ -0,0 +1,79 @@
+# Updating the perl modules repositories in OBS
+
+There are a number of scripts that can update the following repositories:
+* https://build.opensuse.org/project/show/devel:languages:perl:autoupdate
+* https://build.opensuse.org/project/show/devel:languages:perl:CPAN-A (B, C, D
+  etc.)
+
+## Usage
+
+This is how a crontab could look like:
+
+    0 2 * * * $HOME/cpanspec/bin/fetch-cpan.sh > $HOME/obs-mirror/logs/cpan.log 2>&1
+    0 3 * * * $HOME/cpanspec/bin/update-autoupdate.sh > $HOME/obs-mirror/logs/autoupdate.log 2>&1
+    0 4 * * * $HOME/cpanspec/bin/update-dlp-cpan.sh > $HOME/obs-mirror/logs/cpanupdate.log 2>&1
+
+First you need to create `~/obs-mirror` and `~/obs-mirror/logs`.
+
+The `fetch-cpan.sh` will fetch the list of currently indexed modules from CPAN.
+
+`update-autoupdate.sh` will update the
+[autoupdate](https://build.opensuse.org/project/show/devel:languages:perl:autoupdate)
+repository. It will branch projects from
+[`devel:languages:perl`](https://build.opensuse.org/project/show/devel:languages:perl)
+and build the newest version from CPAN.
+
+Requests for submitting the new versions into `devel:languages:perl` have to be
+created manually.
```

Comment:
You should probably elaborate on this a bit more.

Reply:
ok